tests/test_github.py

```diff
@@ -52,6 +53,38 @@ def mock_input(msg: str) -> str:
     return '1'
 
 
+prs_per_page = 100
+number_of_pages = 3
+
+
+def mock_read(self: Any) -> bytes:
+    response_data = [
+        {
+            'head': {'ref': f'feature_{i}', 'repo': {'full_name': 'testing/checkout_prs',
+                                                     'html_url': 'https://github.com/tester/repo_sandbox.git'}},
+            'user': {'login': 'github_user'},
+            'base': {'ref': 'develop'},
+            'number': f'{i}',
+            'html_url': 'www.github.com',
+            'state': 'open'
+        } for i in range(mock_read.counter * prs_per_page, (mock_read.counter + 1) * prs_per_page)]  # type: ignore[attr-defined]
+
+    mock_read.counter += 1  # type: ignore[attr-defined]
+    return json.dumps(response_data).encode()
+
+
+def mock_info(x: Any) -> Dict[str, Any]:
+    if mock_info.counter < number_of_pages - 1:  # type: ignore[attr-defined]
+        link = f'<https://api.github.com/repositories/1300192/pulls?page={mock_info.counter + 2}>; rel="next"'  # type: ignore[attr-defined]
+    else:
+        link = ''
+    mock_info.counter += 1  # type: ignore[attr-defined]
+    return {"link": link}
+
+
+mock_info.counter = mock_read.counter = 0  # type: ignore[attr-defined]
+
+
```

Comment:
I mocked these two functions to simulate 3 page long (with 100 PRs per page) GitHub API response.